addon-resizer/nanny/main/pod_nanny.go

```diff
@@ -73,7 +73,7 @@ func main() {
 	checkPercentageFlagBounds("recommendation-offset", *recommendationOffset)
 	checkPercentageFlagBounds("acceptance-offset", *acceptanceOffset)
 
-	pollPeriod := time.Millisecond * time.Duration(*pollPeriodMillis)
+	pollPeriod := time.Duration(int64(*pollPeriodMillis) * int64(time.Millisecond))
```

Comment:
Why these casts are needed?

Reply:
Without the casts the compiler throws the following error:

invalid operation: *pollPeriodMillis * time.Millisecond (mismatched types int and time.Duration)